test/cram/test.t

```diff
@@ -47,11 +47,11 @@ This is a cram test for the new executable.
   Operation 3 of 14, 1 hunk
   1 addition, 1 removal
   Modification of lib/dune
-  @@ -0,3 +0,3 @@
-    (library
-     (name diffcessible)
-  -  (libraries notty nottui lwd))
-  +  (libraries notty nottui lwd patch))
+  @@ -1,3 +1,3 @@
```

Comment:
It would be nice to have a test case showing a hunk that do not start at line 1 and where "mine" and "their" counts do not start at the same number.
This could be added to as a new .diff file containing just that and shown in this cram test.

Reply:
@Julow Sure! I would read up on .diff files and start working on it. Do you want just one testcase for that in the new .diff file?

Reply:
One testcase would be enough but if you find several testcases that are interesting for different reasons, add them too :)

Reply:
@Julow I have added two test cases: one where the hunk doesn't start at line 1, and one where the hunk doesn't start at line 1 and the "mine" and "their" counts don't start at the same line. Please review them.